test/crux/api_test.clj

```diff
@@ -17,6 +17,13 @@
 
 (declare execute-sparql)
 
+(t/deftest test-content-hash-invalid
+  (let [valid-time (Date.)
+        content-ivan {:crux.db/id :ivan :name "Ivan"}
+        content-hash (c/new-id content-ivan)]
```

Comment:
See above, would be nice with a sanity check assertion that the confromed message contains the hash.